core/commands/publish.go

```diff
@@ -84,20 +82,14 @@ Publish an <ipfs-path> to another public key (not implemented):
 			pstr = args[0]
 		}
 
-		node, err := n.Resolver.ResolvePath(path.FromString(pstr))
+		p, err := path.ParsePath(pstr)
 		if err != nil {
-			res.SetError(fmt.Errorf("failed to resolve path: %v", err), cmds.ErrNormal)
-			return
-		}
-
-		key, err := node.Key()
-		if err != nil {
-			res.SetError(err, cmds.ErrNormal)
```

Comment:
this part should still verify that indeed this path resolves to an object. It will help people not publish mistaken things. (could have a `-f` if really want to be able to publish something that isn't resolving to an object atm)